packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -135,4 +136,97 @@ export default class ScheduledPaymentModule {
 
     return { txs: [transaction, setGuardTransaction], expectedModuleAddress };
   }
+
+  async generateSchedulePaymentSignature(
```

Comment:
This is in a separate function because this function will be used by the web client to get the signature, and then send it to the hub so that it will be able to add the spHash to the module.